src/Autosuggest.js

```diff
@@ -78,11 +79,13 @@ class Autosuggest extends Component {
 
   componentWillReceiveProps(nextProps) {
     if (nextProps.suggestions !== this.props.suggestions) {
-      const { suggestions, inputProps, shouldRenderSuggestions,
-              isCollapsed, revealSuggestions, lastAction } = nextProps;
+      const {
+        suggestions, inputProps, shouldRenderSuggestions, isCollapsed,
+        revealSuggestions, lastAction, alwaysRenderSuggestions
+      } = nextProps;
       const { value } = inputProps;
 
-      if (isCollapsed && lastAction !== 'click' && lastAction !== 'enter' &&
+      if (alwaysRenderSuggestions || isCollapsed && lastAction !== 'click' && lastAction !== 'enter' &&
```

Comment:
Do we really need `alwaysRenderSuggestions ||` here? 
All tests pass without it, so I wonder if we can just remove it, or are we missing a test here?